Gemfile

```diff
@@ -98,6 +98,7 @@ group :production do
 end
 
 group :test do
+  gem "action-cable-testing"
```

Comment:
Added this gem because there aren't ways to test ActionCable channels currently in Rails. This gem is also currently a PR on Rails/Rails so it's probably our best option.

Should I set a static version?

Reply:
:+1: , and yeah lets specify a minor version here.

`"~> 0.3"`